packages/haul-cli/src/commands/start.ts

```diff
@@ -87,19 +87,19 @@ export default function startCommand(runtime: Runtime) {
       }
 
       const configPath = getProjectConfigPath(directory, argv.config);
-      const projectConfig = getNormalizedProjectConfigBuilder(configPath)(
+      const projectConfig = getNormalizedProjectConfigBuilder(
         runtime,
-        {
-          platform: '',
-          root: directory,
-          dev: argv.dev,
-          port: argv.port,
-          bundleMode: 'multi-bundle',
-          bundleTarget: 'server',
-          assetsDest: tempDir,
-          minify: argv.minify === undefined ? !argv.dev : argv.minify,
-        }
-      );
+        configPath
+      )(runtime, {
```

Comment:
why passing runtime again? it could be injected from the closure

Reply:
I could but it would be more problematic to type this, since I'll need to add another type: not only I would need `NormalizedProjectConfigBuilder` but also `NormalizedProjectConfigBuilderPartialApplied` os something which would not require `runtime` parameter. IMO it's unnecessary.